server_cmds.go

```diff
@@ -340,14 +340,142 @@ func (s *ServerMethods) PrivilegeKeyAdd(ttype, id1, id2 int, options ...CmdArg)
 
 // OnlineClient represents a client online on a virtual server.
 type OnlineClient struct {
-	ID          int    `ms:"cid"`
+	ID          int    `ms:"clid"`
+	ChannelID   int    `ms:"cid"`
 	DatabaseID  int    `ms:"client_database_id"`
 	Nickname    string `ms:"client_nickname"`
 	Type        int    `ms:"client_type"`
 	Away        bool   `ms:"client_away"`
 	AwayMessage string `ms:"client_away_message"`
 }
 
+// Like OnlineClient but with much more information.
+type DetailedOnlineClient struct {
+	OnlineClient `ms:",squash"`
+
+	// Milliseconds since the client did something, for example sending
+	// a message, muting themselves or talking.
+	IdleTime int64 `ms:"client_idle_time"`
```

Comment:
Could we capture this as a time.Duration, so we don't have to know its in Milliseconds?